2022/day_6.py

```diff
@@ -0,0 +1,64 @@
+#%% example signal
+
+def find_marker(signal_string, marker_length):
+    for s in range(marker_length, len(signal_string) + 1):
+        start = s - marker_length
+        stop = s
+        target = signal_string[start : stop]
```

Comment:
No need to make a new variable for `stop`
```suggestion
        target = signal_string[start : s]
```